### PR TITLE
Tenant UI - Simple Form

### PR DESCRIPTION
**Pull request by popkinj**
## Bare Bones Tenant UI
- Form to enter Wallet ID and Key
- Submit and clear buttons
- State to deactivate form during async request to API
- Requesting token from API
- Error handling failed requests
- Displaying token on screen if successful

## Development Environment
- Vite/Rollup stock development environment
- Typescript
- Docker config building production version of app
- Docker-compose integration with the rest of the project

## Actions
- I'll rebase after I open the PR
- Should I have the docker-compose port number as an Environment Variable?

### CODE REVIEW

File: services/traction/api/core/config.py [hunk 1]
```diff
@@ -88,6 +88,9 @@ class GlobalConfig(BaseSettings):
 
     DEFAULT_PAGE_SIZE: int = os.environ.get("DEFAULT_PAGE_SIZE", 10)
 
+    # Allow CORS from a comma-separated list of origins
+    TRACTION_CORS_URLS: str = os.environ.get("TRACTION_CORS_URLS", "")
```
Comment (loneil) [inline services/traction/api/core/config.py:92]: Think we should add this to env-example so people can just set it when they need it after copying over the example.
See how we do SHOWCASE_CORS_URLS in .env-example

**Review by loneil — CHANGES_REQUESTED** [review]
Probably just an additon of that env var to the example file

File: services/traction/api/core/config.py (hunk 1 above)
Comment (popkinj) [reply services/traction/api/core/config.py:92]: Good thinking.

**Comment by popkinj** [comment]
Tenant UI bar bones setup. Vue3, Vite & PrimeVue. Docker config incorporated into Docker-Compose.

**Pull request merged**